### ISSUE TITLE
Feature

### ISSUE THREAD

**Issue by legroen** [v-zhuravlev/zabbix-notify#31]
Hello,

I see that there is a possibility to attach the acknowledgements as replies to the problem message. Will it be possible in the future to add the recovery message as a reply to the problem message?

**Comment by v-zhuravlev** [comment]
Yes, it is possible to add in the future. In the meantime, there is a new mode `--slack_mode=alarm-no-delete` that can be useful to you?

**Comment by dcms-world** [comment]
We've also the problem that there are very much Messages in Slack Channels, it would be easier if the OK Messages are in the thread from the respective Problems in order to get the Channels clearer
